apps/client/pages/login.tsx

```diff
@@ -28,15 +28,15 @@ const Login: React.FC = () => {
           password: values.password,
         },
       },
-    }).catch((caughtError) => {
+    }).catch((error) => {
       setLoginError('Wrong email or password!');
       setTimeout(() => {
         setLoginError('');
       }, ERROR_MODAL_TIMEOUT);
     });
     if (response) {
       // Set authorization cookie to response token (if we are working at different domains and it's not set automatically) THIS IS REALLY UNWANTED
-      
+
       response.data.login.introductionCompleted
```

Comment:
Probably need `?` in there

Reply:
it's in the next line

Reply:
I've meant we need to put `response?.data?.login?.introductionCompleted` instead of `response.data.login.introductionCompleted`

Reply:
As we can have a response without data an many more errors on the way, which would throw off an app :)